recommendations/index.md

```diff
@@ -135,6 +135,45 @@ Furthermore, multiple filters can be applied to a single request:
 GET /comments?filter[post]=1,2&filter[author]=12 HTTP/1.1
 ```
 
+## <a href="#including-links" id="including-links" class="headerlink"></a> Including Top-level, Resource-level and Relationship Links
+
+The base specification is agnostic about including links with a resource response. However it is recommended that `links` the following links are always included within the response for a resource.
+
+- **Top-level links** like a self-link (for the whole response) as well as relative pagination links (if appropriate).
+- **Resource-level links** like a self-link for each resource (which differs from the top-level, if the resource is part of a collection).
+- **Relationship links** for all available relationships of a resource.
+
+For example, the following request response for a comment returns all recommended links:
+
+```http
+GET /comments/1 HTTP/1.1
+
+{
```

Comment:
This example needs a `data` element and a top-level links (like the home page example) - probably best illustrated with a `GET` to a collection.